bin/wrapper_region_sbatch.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
might be worth not having these hard-coded, but it's probably not a big deal either.